src/main/resources/alma/fix/mediumAndType.fix

```diff
@@ -558,8 +558,20 @@ if any_match("@leaderTyp+008", "(Continuing Resources)(.{21})p.*")  # Pos21
   add_field("type[].$append","Periodical")
 elsif any_match("006", "^[s](.{3})p.*") # Pos00+04
   add_field("type[].$append","Periodical")
-elsif any_match("natureOfContent[].*.label", ".*(Zeitschrift|Werkzeitschrift).*")
+elsif any_match("natureOfContent[].*.label", ".*(Zeitschrift|Werkzeitschrift|eriodical).*")
```

Comment:
Was this intentional or is the "P" missing?

Reply:
This is intentional to catch Periodical and periodical.

Reply:
Maybe make it explicit then? `[Pp]`

Reply:
thanks for the suggestion went with `(?i).*(Zeitschrift|Periodical).*`